app/src/ui/diff/image-diffs/two-up.tsx

```diff
@@ -27,40 +29,68 @@ export class TwoUp extends React.Component<ITwoUpProps, {}> {
+      this.props.current.bytes
+    )
+    const diffBytes = this.props.current.bytes - this.props.previous.bytes
 
     return (
-      <div className="image-diff-two-up" ref={this.props.onContainerRef}>
-        <div className="image-diff-previous">
-          <div className="image-diff-header">Deleted</div>
-          <ImageContainer
-            image={this.props.previous}
-            onElementLoad={this.props.onPreviousImageLoad}
-            style={style}
-          />
+      <div className="image-diff-container" ref={this.props.onContainerRef}>
+        <div className="image-diff-two-up">
+          <div className="image-diff-previous">
+            <div className="image-diff-header">Deleted</div>
+            <ImageContainer
+              image={this.props.previous}
+              onElementLoad={this.props.onPreviousImageLoad}
+              style={style}
+            />
 
-          <div className="image-diff-footer">
-            <span className="strong">W:</span> {previousImageSize.width}px |{' '}
-            <span className="strong">H:</span> {previousImageSize.height}px
+            <div className="image-diff-footer">
+              <span className="strong">W:</span> {previousImageSize.width}px |{' '}
+              <span className="strong">H:</span> {previousImageSize.height}px |{' '}
+              <span className="strong">Size:</span>{' '}
+              {formatBytes(this.props.previous.bytes)}
+            </div>
           </div>
-        </div>
 
-        <div className="image-diff-current">
-          <div className="image-diff-header">Added</div>
-          <ImageContainer
-            image={this.props.current}
-            onElementLoad={this.props.onCurrentImageLoad}
-            style={style}
-          />
+          <div className="image-diff-current">
+            <div className="image-diff-header">Added</div>
+            <ImageContainer
+              image={this.props.current}
+              onElementLoad={this.props.onCurrentImageLoad}
+              style={style}
+            />
 
-          <div className="image-diff-footer">
-            <span className="strong">W:</span> {currentImageSize.width}px |{' '}
-            <span className="strong">H:</span> {currentImageSize.height}px
+            <div className="image-diff-footer">
+              <span className="strong">W:</span> {currentImageSize.width}px |{' '}
+              <span className="strong">H:</span> {currentImageSize.height}px |{' '}
+              <span className="strong">Size:</span>{' '}
+              {formatBytes(this.props.current.bytes)}
+            </div>
           </div>
         </div>
+        <div className="image-diff-summary">
+          Diff:{' '}
+          <span
+            className={classNames({
+              added: diffBytes > 0,
+              removed: diffBytes < 0,
+            })}
+          >
+            {diffBytes !== 0
+              ? `${formatBytes(diffBytes)} | ${diffPercent}`
+              : 'No size difference'}
```

Comment:
Thoughts on adding a plus/minus sign to indicate if there was a positive or negative change in size? (For example `+2KB | +26%` or `-2KB | -26%`) It'll help make the number a little quicker to parse, especially for those with red/green color blindness.

![screen shot 2018-04-30 at 10 19 25 am](https://user-images.githubusercontent.com/1174461/39441143-d9d86e4c-4c61-11e8-8d54-a7a3685d5b11.png)

Reply:
>  It'll help make the number a little quicker to parse, especially for those with red/green color blindness.

Big 👍 for this point

Reply:
Thanks! That was my idea as well, just slipped through during some code refresh, should be added properly now.